volatility3/framework/plugins/linux/threads.py

```diff
@@ -0,0 +1,67 @@
+# This file is Copyright 2019 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+from typing import Callable, Iterable, List, Any
+
+from volatility3.framework import renderers, interfaces
```

Comment:
In PEP8, it is recommended that modules be sorted alphabetically.

```suggestion
from typing import Any, Callable, Iterable, List

from volatility3.framework import interfaces, renderers
```

Reply:
Minor - but I agree to commit the suggestion.